react-toggle-switch-component/my-toggle-button/src/ToggleSwitch.jsx

```diff
@@ -1,8 +1,9 @@
 import { useState } from 'react';
 import './ToggleSwitch.css';
 
-export default function ToggleSwitch({ label }) {
+export default function ToggleSwitch() {
   const [toggled, setToggle] = useState(true);
+  let label;
```

Comment:
Good. I prefer to use a tenary operator and set the label in one statement. That also lets me declare `label` to be a const:
`const label = toggled ? 'ON' : 'OFF';`